editorial pass packet scheduling section (#531)
quicwg/multipath: draft-ietf-quic-multipath.md

## Packet Scheduling {#packet-scheduling}

The transmission of data packets is limited

> by the arrival of data from the application and the congestion control

Author: ```suggestion
by the arrival of data from the application and by congestion control.
```

> window. Generally, QUIC packets that increase the number of bytes in flight can only be sent

Reviewer: Also, the transmission of stream data is subject to flow control. Don't know whether we need to mention that.

when the congestion window for the selected path allows it.

Most frames, including control frames (PATH_CHALLENGE and PATH_RESPONSE being the notable
exceptions), can be sent and received on any open path.
As such a packet scheduler is needed to decide which path to use
for sending the next packet, among those paths with an open congestion window.
The scheduling is a local decision, based on the preferences of the application and the
implementation.

This implies that an endpoint may send and receive PATH_ACK
frames on a path different from the one that carried the acknowledged
packets. As noted in {{compute-rtt}}, RTT estimates computed using
the standard algorithm reflect both the characteristics of the
path and the scheduling algorithm of PATH_ACK frames. The estimates will converge

> faster if the scheduling strategy is stable.

Reviewer: I would specify "the scheduling strategy of PATH_ACK frames is stable."

Author: ```suggestion
faster if the scheduling strategy of PATH_ACK frames is stable.
```

> faster if the scheduling strategy is stable. 

Author: ```suggestion
faster if the scheduling strategy is stable.
```

> Still, implementations can choose different strategies such as sending

Reviewer: Delete "Still, ".

Author: ```suggestion
Implementations can choose different strategies such as sending
```

> Implementations can choose different strategies such as, for instance, sending

Reviewer: I would write "Different implementations can choose different strategies", to emphasize that the difference we speak about here is a differencece between implementations and not a difference between the implementation and some kind of generic behavior.

> Implementations can choose different strategies such as sending
> PATH_ACK frames e.g. either simply on the path where the acknowledged packets was received,

Reviewer 2: The e.g. in the middle of the sentence is very strange.
```suggestion
Implementations can choose different strategies such as, for instance, sending
PATH_ACK frames either simply on the path where the acknowledged packets was received,
```

or alternatively the shortest path, which results in shorter control loops
and potentially better performance.

Since packets that only carry PATH_ACK frames
are not congestion controlled (see {{Section 7 of QUIC-RECOVERY}}),
senders should carefully consider the load induced

> by these packets, especially if the capacity is unknown on that path
> e.g when that path is not used for sending data frames.

Reviewer 2: ```suggestion
by these packets, especially if the capacity is unknown on that path,
e.g., when that path is not used for sending data frames.
```
